node/src/reactor/joiner.rs

```diff
@@ -823,7 +823,7 @@ impl reactor::Reactor for Reactor {
 
             Event::LinearChainAnnouncement(LinearChainAnnouncement::BlockAdded {
                 block_hash,
-                block_header,
+                block: block_header,
```

Comment:
This is not clear: is it a block or a block_header?

Reply:
It should be block_header